### ISSUE TITLE
Noah LSM Version Used by WRF

### ISSUE THREAD

**Issue by GGDLW** [wrf-model/WRF#2079]
Hello,

I am here today to find out which version of the Noah LSM that the WRF model uses. I was unsure which README document would contain this information. I want to be sure that the version of Noah that the LIS model is running is the same as the version of the Noah LSM that WRF is running.

**Comment by weiwangncar** [comment]
@cenlinhe Can you comment, if you know? Thanks!

**Comment by cenlinhe** [comment]
Unfortunately, I am not sure about the Noah version in WRF, but I guess the LIS Noah version is the Noah version in WRF v3.9. It is a question for David Mocko from LIS team who I believe implemented the Noah v3.9 to LIS.

**Comment by GGDLW** [comment]
I am currently using LIS 7.3, which had Noah version 3.6, does that mean that the version of WRF I am using (4.5) also uses Noah version 3.6?

**Comment by cenlinhe** [comment]
No, I do not think so. LIS includes both Noah v3.9 and Noah v3.6. So the Noah version in LIS was probably determined by the WRF version.

**Comment by GGDLW** [comment]
I got the news that WRF version 4.5.2 uses Noah LSM version 5.0 according to this thread: https://forum.mmm.ucar.edu/threads/noah-lsm-version.18191/

**Comment by cenlinhe** [comment]
Noah is different from Noah-MP. Noah-MP is now version 4.5 in WRFv4.5.2, and Noah-MP v5 is going to be coupled with WRF for the next year's major release.

**Comment by GGDLW** [comment]
Would it make sense for the Unified Noah LSM (sf_surface_physics option 2) to be the same version (4.5) as the Noah Multi-Parameter (sf_surface_physics option 4) in that case?

**Comment by cenlinhe** [comment]
The versioning of Unified Noah LSM is not very clear. Previously, people always use the same WRF version for the Unified Noah LSM version, but there is no official document or versioning for Noah now because we have stopped updating the Noah code (unless users want to include any physics update or bug fixes).
